src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -123,17 +123,21 @@ def __init__(self, msConfig, mode, logger=None):
         msOutIndex = IndexModel('RequestName', unique=True)
         msOutDBConfig = {
             'database': self.msConfig['mongoDB'],
-            'server': self.msConfig['mongoDBUrl'],
+            'server': self.msConfig['mongoDBServers'],
+            'replicaset': self.msConfig['mongoDBReplicaset'],
             'port': self.msConfig['mongoDBPort'],
+            'username': self.msConfig['mongoDBUser'],
+            'password': self.msConfig['mongoDBPassword'],
+            'connect': True,
+            'directConnection': False,
```

Comment:
Todor, can you add a short comment on what this `connect` and `directConnection` parameters are?

Reply:
From the `MongoClient` documentation again:
```
"""
...
         - `connect` (optional): if ``True`` (the default), immediately
            begin connecting to MongoDB in the background. Otherwise connect
            on the first operation.
         - `directConnection` (optional): if ``True``, forces this client to
            connect directly to the specified MongoDB host as a standalone.
            If ``false``, the client connects to the entire replica set of
            which the given MongoDB host(s) is a part. If this is ``True``
            and a mongodb+srv:// URI or a URI containing multiple seeds is
            provided, an exception will be raised.
"""
```

Reply:
Good idea. It would be helpful also in the comment few lines above. I am adding this now.